src/config/cfg_rules.ini

```diff
@@ -11,7 +11,8 @@ section = ifp
 section = secrets
 section = kcm
 section_re = ^secrets/users/[0-9]\+$
-section_re = ^domain/.*$
+section_re = ^domain/[^/\@]\+$
+section_re = ^domain/[^/\@]\+/[^/\@]\+$
 section_re = ^application/.*$
 
```

Comment:
Is there a reason why application domain allows all characters?
Because domain section allows everything excluding `/` and `@`